epsagon/trace.py

```diff
@@ -1012,6 +1047,9 @@ def _send_traces(self):
             if self.runner:
                 self.runner.terminate()
 
+                if self.log_id:
+                    self.runner.resource['metadata']['log_id'] = self.log_id
```

Comment:
we need to put it not under 'metadata', it's an internal field of ours

Reply:
I dont under stand, this is `metadata`